### PR TITLE
Oh, the unbearable burden we the groovy QA people must bear!

### PR DESCRIPTION
**Pull request by QAscoop**
'Be kind for everybody you meet is fighting a hard battle.'

### CODE REVIEW

**Comment by jmeng30** [comment]
💯

**Comment by TranForsythe** [comment]
Pros: You've thoroughly defined your priorities in life.
Cons: Your priorities in life... stink. ;)

File: GroovyQA_Mehmet-burbank.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Burbank, CA
+
+### Places to Smoke
+
```
Comment (TranForsythe) [inline GroovyQA_Mehmet-burbank.md:4]: Maybe fill in... at least one?

File: GroovyQA_Mehmet-burbank.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+### Places to NOT-Smoke
```
Comment (jmeng30) [inline GroovyQA_Mehmet-burbank.md:5]: Where you are not supposed to smoke!

File: GroovyQA_Mehmet-burbank.md (hunk 1 above)
Comment (QAscoop) [reply GroovyQA_Mehmet-burbank.md:4]: Parking Lot?

**Pull request merged**